apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/metadata.json

```diff
@@ -1,5 +1,7 @@
 {
   "resourceType" : "CapabilityStatement",
+  "id" : "IGNORED_FIELD",
```

Comment:
this is legitimate -- the ITs perform this substitution only when the value matches the UUID pattern which in this case it does since a new random UUID is populated every time this is called.